### ISSUE TITLE
CLI: Get data

### ISSUE THREAD

**Issue by VeckoTheGecko** [OceanParcels/virtualship#68]
`virtualship fetch`: Downloads the data needed for the simulation as an explicit step so that users can do it separately. If `virtualship` is ran without this, then it will go and fetch this data.

**Comment by VeckoTheGecko** [comment]
Also see if we can use one environment for everything:

```
Data can be downloaded with the `download_data.py` script. For now a different conda env is needed for downloading, see comments in the script.
```

**Comment by iuryt** [comment]
Is someone working on this? If not, I can try to help.
Should we add new variables to schedule, such as `bbox` for the `area_of_interest`?
The rest would be to basically adapt `scripts/download_data.py` to here https://github.com/OceanParcels/virtualship/blob/6f843e969f51cbe6cce0aa374889d3a9b0e49693/src/virtualship/cli/commands.py#L48-L50

Let me know your thoughts.

**Comment by iuryt** [comment]
Yes, I lean more towards adding this to the config file. For example, if we have an experiment that relies only on drifters, the study area and time range will probably be considerably larger than the schedule.

If you haven't started working on this, I can initiate a PR and we can collaborate from there. What do you think?

**Comment by VeckoTheGecko** [comment]
@iuryt Sounds good! Happy for you to do a PR and I can jump in in review

**Comment by ammedd** [comment]
The download_data is all I used before. It was based on an area of interest based on the sampling stations, but indeed, it would be nice to extend this a bit. And in case of Argo/drifter deployment It downloaded a separate dataset that had an area 3degrees more on each side to allow for 3 additional weeks of data from the deployed instruments.

**Comment by ammedd** [comment]
Closed with #83
